packages/db/src/index.ts

```diff
@@ -59,6 +59,20 @@ export default (req: Request, res: Response, next: NextFunction) => {
 
     rr.onBefore(async (params) => {
       const { zencode, keys, data } = params;
+      keysContent =
```

Comment:
Kind of cryptic, don't you want to move this duplicated logic inside the `parse` function?

so you can write:
```js
 content = {...parse(data), ...parse(keys)} 
```

would become more readable and maintainable

Reply:
This one slipped through! Made the adjustments as suggested.